index.html

```diff
@@ -441,6 +442,37 @@ <h3>Integrity Validation</h3>
           <p class="issue" data-number="143">
             Provide examples for Data Integrity and VC-JWT Credential Schemas
           </p>
+          <p>
+            The integrity of Credential Schemas of type <a href="#verifiablecredentialschema2023">JsonSchema2023</a> MAY
```

Comment:
this should be amended to apply to both `JsonSchema2023` and `VerifiableCredentialSchema2023`, other examples should be updated too.

Reply:
`VerifiableCredentialSchema2023` already includes integrity guarantees being that it's a secured message. For what use cases are you thinking this would be useful?